src/pytkdocs/loader.py

```diff
@@ -741,11 +741,19 @@ def get_django_field_documentation(node: ObjectNode) -> Attribute:
         if prop.blank:
             properties.append("blank")
 
+        # set correct docstring based on verbose_name and help_text
+        # both should be converted to str type in case lazy translation
+        # is being used, which is common scenario in django
+        if prop.help_text:
+            docstring = "`{}`: {}".format(prop.verbose_name, prop.help_text)
```

Comment:
Lets not assume Markdown:
```suggestion
            docstring = f"{prop.verbose_name}: {prop.help_text}"
```